projects/com.oracle.truffle.llvm.nodes/src/com/oracle/truffle/llvm/nodes/base/LLVMTerminatorNode.java

```diff
@@ -40,21 +39,14 @@
  * @see <a href="http://llvm.org/docs/LangRef.html#terminator-instructions">terminator
  *      instructions</a>
  */
-public abstract class LLVMTerminatorNode extends LLVMNode {
+public abstract class LLVMTerminatorNode extends LLVMControlFlowNode {
 
-    @CompilationFinal private final int[] successors;
+    @CompilationFinal(dimensions = 1) private final int[] successors;
```

Comment:
Having no `dimensions` defaults to the array dimensions of the field type. So there is no change in behavior.